frontend/src/components/payment/DonationInput.module.scss

```diff
@@ -0,0 +1,42 @@
+.form {
+  display: flex;
+  flex-direction: column;
+
+  background-color: var(--bg-color-secondary);
+  border-radius: var(--border-radius);
+
+  gap: 20px;
+  padding: 20px;
+  margin: 20px 0;
+
+  h4 {
+    margin: 0;
+  }
+
+  .options {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+
+    gap: 20px;
+  }
+
+  .amount_input {
```

Comment:
If I remember the docs correct, they recommend `amountInput` for these for some reason